### ISSUE TITLE
Finalise Transcriptions/Records Page

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#305]

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds ,

This will be done using a FreeBMD2 version of this page and various sub-pages, but we will hit a number of problems: many of the final pages that **do something** are Perl pages running on FreeBMD that get graphs or pull things off the data-base. I don't think we have replacements yet on FreeBMD. So, I either have to leave these links **undone** , or I put in (opens in new page)-links to the relevant pages FreeBMD). If I do that were will be a **clash of corporate images**: users will have FreeBMD(1) and FreeBMD2 pages open in different windows (a bit like the current Tab).  I will complete the pages and send them for signing off, but someone at that stage will need to make the final decision: broken links or mixed FreeBMD(1) & 2 pages.

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds ,
 I have a draft page at https://test3.freebmd.org.uk/cms/draft-coverage on Refinery. (Note: the label "Transcriptions" has now changed(?) to "Coverage", I beleive).

This draft page is more of an expansion of the corresponding FreeBMD(1) page, rather than say  a reworking of the FreeCEN & FreeREG Coverage pages, with just their tables with hyperlinks to their charts and "places" (Registration District or Parish, etc), etc; but the equivalent ones for FreeBMD2 would appear at the level below this (draft) page.

Most of what is being described does not yet exist on FreeBMD(2) and that includes "signed off" versions of all the html pages that are referred to in the outline hyperlinks; and, for example, there does not appear to be any Free- -REG  / -CEN equivalent of https://www.freebmd.org.uk/db-latest.shtml , nor the bulk (i.e. Submitters) of https://www.freebmd.org.uk/db-latest.shtml .

At the moment the pages "below" (hyperlinked to) this daft are being worked on, and where these are likely to impact on this one it is being amended. Not having seen this page for a couple of weeks or so, I think it is "too wordy", so its likely to be cut back.
Comments would be welcome.

**Comment by richpomfret** [comment]
Changed title to reflect no current need for this at MVP.

**Comment by KristinaGadzhieva** [comment]
@richardofsussex @DeniseColbert can we close this story?

**Comment by richardofsussex** [comment]
I think we have achieved what we want to for MVP - closing.
